lib/unimatrix/authorization/client_credentials_grant.rb

```diff
@@ -25,14 +25,18 @@ def request_token
           if response.code == '200'
             body = JSON.parse( response.body )
             body = body[ 'token' ] if body[ 'token' ].present?
-
-            body[ 'access_token' ] rescue nil
+            result = body[ 'access_token' ] rescue nil
+            if options == 'include_expiry' 
```

Comment:
I would make options a hash (I think this is somewhat standard?) so that it's easy to add other options in the future. So you could access them by doing `if options['include_expiry'] == true` or something like that.